beacon-chain/simulator/service.go

```diff
@@ -51,29 +52,19 @@ type Config struct {
 	BlockRequestBuf     int
 	BlockSlotBuf        int
 	ChainHeadRequestBuf int
-	CStateReqBuf        int
+	StateReqBuf         int
 	P2P                 p2pAPI
 	Web3Service         powChainService
-	BeaconDB            beaconDB
+	BeaconDB            *db.BeaconDB
 	EnablePOWChain      bool
 }
 
-type beaconDB interface {
```

Comment:
Why was this removed ? We would only want to expose this service to methods that it will actually use.